modules/core/src/passes/layers-pass.js

```diff
@@ -1,10 +1,13 @@
 import GL from '@luma.gl/constants';
 import Pass from './pass';
-import {clear, setParameters, withParameters} from '@luma.gl/core';
+import {clear, setParameters, withParameters, Framebuffer} from '@luma.gl/core';
 
 export default class LayersPass extends Pass {
-  render(props) {
-    return this.drawLayers(props);
+  render(params) {
+    const gl = this.gl;
+    const {outputBuffer = Framebuffer.getDefaultFramebuffer(gl)} = params;
+
+    return withParameters(gl, {framebuffer: outputBuffer}, () => this.drawLayers(params));
```

Comment:
Can we not change the render behavior when no postProcessEffects are used?

Reply:
when there is no PostProcessEffects, the outputBuffer is the default frame buffer(screen), the behavior is the same as the old logic, the benefit with this solution is that LayersPass don't need know about which surface for it to render.